### PR TITLE
Add new flags

### PR DESCRIPTION
**Pull request by nabaroa**
I had doubts on xof and xaf cases, because it's a currency of a group of countries: https://en.wikipedia.org/wiki/West_African_CFA_franc, https://en.wikipedia.org/wiki/Central_African_CFA_franc

### CODE REVIEW

**Comment by nabaroa** [comment]
Designed default flag for special cases. In this case 'xof' and 'xaf'
image

**Comment by matallo** [comment]
LGTM, loving the placeholder flag!

something that made me think is having the flags base64 encoded in the CSS could be a blocker in the page load (I've read this being an issue somewhere else), besides that 👍 

@nabaroa do you think an external asset (it can well be a sprite as them all flags will load at once) is worth it?

**Comment by oliverviljamaa** [comment]
@matallo We had a ton of problems with our sprites in the past, hence this repo, _but_ it'd be time to revisit it as there's definitely a number of new tools that have appeared since... 2 years or something 😀And we now have _way_ more flags here.

**Comment by nabaroa** [comment]
The placeholder flag is double checked and send by designers, but Oliver I will send them your feedback. Thanks

**Comment by oliverviljamaa** [comment]
@nabaroa Maybe the designers are not aware that this is an open-source package not consumed by us only?

**Comment by nabaroa** [comment]
Fair point. I will change it :rocket:

**Comment by nabaroa** [comment]
Choosed a more generic default flag
image

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "currency-flags",
-  "version": "1.2.1",
+  "version": "1.3.1",
```
Comment (oliverviljamaa) [inline package.json]: Should be `1.3.0`, right?

Comment (nabaroa) [reply package.json]: Right

**Pull request merged**